### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by WBerredo**
Setup of:

* package.json
* Babel
* Webpack
* Eslint
* Unit Tests (Mocha, Karma, Chai)
* standard-version

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "parserOptions": {
+    "ecmaVersion": 2017,
+    "ecmaFeatures": {
+      "experimentalObjectRestSpread": true
+    }
+  },
+  "env": {
+    "node": true
```
Comment (andrewcpacifico) [inline .eslintrc]: acho que o env aqui é browser.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "parser": "babel-eslint",
```
Comment (andrewcpacifico) [inline .eslintrc]: Why this option necessary?

File: karma.conf.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+module.exports = function(config) {
```
Comment (andrewcpacifico) [inline karma.conf.js]: It's missing the webpack config. Have you not corrected this?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "banner-sdk",
```
Comment (andrewcpacifico) [inline package.json]: @linx/banner-client-js

File: package.json [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      "name": "Wesley Berrêdo",
+      "email": "wberredo@gmail.com"
+    }
+  ],
+  "files": [
+    "dist"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/WBerredo/banner-sdk/issues"
```
Comment (andrewcpacifico) [inline package.json]: Update this url.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  },
+  "homepage": "https://github.com/WBerredo/banner-sdk#readme",
```
Comment (andrewcpacifico) [inline package.json]: Update this one too.

File: webpack.common.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    libraryTarget: 'umd',
+    library: 'LinxBanner',
```
Comment (andrewcpacifico) [inline webpack.common.js]: Check if it is possible to export to `window.linx.banner`. If it is not possible, I think that it is better to do it manually on the `index.js`.

Something like:

```javascript
import { module };

const LinxBanner = {
  module,
};

window.linx = window.linx || {}:
window.linx.banner = LinxBanner;
```

File: webpack.common.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+      },
+      {
+        enforce: 'pre',
```
Comment (andrewcpacifico) [inline webpack.common.js]: what is this?

Comment (WBerredo) [reply webpack.common.js]: [This rule](https://webpack.js.org/configuration/module/#rule-enforce) tell which block should come first. In this case, it tells eslint should be executed before code generation by babel.

File: .eslintrc (hunk 2 above)
Comment (WBerredo) [reply .eslintrc]: it's not :D
I got by another project, but I will fix it.

File: karma.conf.js (hunk 3 above)
Comment (WBerredo) [reply karma.conf.js]: It is on #2 .

File: .eslintrc (hunk 1 above)
Comment (WBerredo) [reply .eslintrc]: It is on #2 .

**Pull request merged**